### ISSUE TITLE
A better way to eliminate confusion

### ISSUE THREAD

**Issue by KennyFu1997** [mml-book/mml-book.github.io#502]
**Describe the mistake**
A better replacement of symbol of a fomula

**Location**
Please provide the 
1. Draft (2020-03-07)
2. Part 1 2.4.2 Vector Spaces
3. page 37
4. Definition 2.9(Vector Space)

**Proposed solution**
2. ∀ λ, ψ ∈ R, x ∈ V : (λ + ψ) · x = λ · x + ψ · x
Here are two operations "+". We know that the first "+" refers to general addition like 3+4, 1+2, but the second "+" refers the to the operation that has been defined in V = (V, +, · ). Readers will consider that the two "+" operations are the same. So I advise to replace the operation "+" in V = (V, +, · ) to another operation that looks different to "+" such as ⊗. Hence this line "∀ λ, ψ ∈ R, x ∈ V : (λ + ψ) · x = λ · x + ψ · x " will be "∀ λ, ψ ∈ R, x ∈ V : (λ + ψ) · x = λ · x ⊗ ψ · x". Obviously readers will understand this formula without any confusion.

**Additional context**
No.

**Comment by mpd37** [comment]
I agree, this would reduce confusion. It's not a super-critical fix and would require more than cosmetic change. 

I'll leave this as a feature request for a future version.
